### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by stefano81**

### CODE REVIEW

File: fluidos_model_orchestrator/common.py [hunk 1]
```diff
@@ -21,11 +20,11 @@ class Resource:
     id: str
     cpu: str = None
     memory: str = None
-    architecture: Optional[str] = None
-    gpu: Optional[str] = None
-    ephemeral_storage: Optional[str] = None
-    persistent_storage: Optional[str] = None
-    region: Optional[str] = None
+    architecture: str | None = None
+    gpu: str | None = None
```
Comment (killianlevacher) [inline fluidos_model_orchestrator/common.py:24]: Hi Stefano, 

just out of curiosity, I've never seen this syntax `str | None` I'm assuming it's better than Optional[str] but was wondering what was the rational for the change?

File: fluidos_model_orchestrator/model/candidate_generation/__init__.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import torch.nn.functional as F
 import json
 import random  # TODO remove
-import importlib_resources
+import pkg_resources  # noqa
```
Comment (killianlevacher) [inline fluidos_model_orchestrator/model/candidate_generation/__init__.py:7]: I'm assuming you're removing importlib due to the package conflict right? The whole PR looks good to me in any case :-)

File: fluidos_model_orchestrator/common.py (hunk 1 above)
Comment (stefano81) [reply fluidos_model_orchestrator/common.py:24]: I thought it looks better. I've no strong evidence that a notation is better than the other. Actually, there would be also `architecture: Union[str, None]` as additional option.

File: fluidos_model_orchestrator/model/candidate_generation/__init__.py (hunk 2 above)
Comment (stefano81) [reply fluidos_model_orchestrator/model/candidate_generation/__init__.py:7]: Yes, unfortunately it seems importlib_resources is somehow not loaded properly. I opened an issue to remind the technical debt of using pkg_resources instead.

**Pull request merged**